ros2topic/ros2topic/verb/hz.py

```diff
@@ -0,0 +1,315 @@
+# Software License Agreement (BSD License)
```

Comment:
Can you please move line 1 and 2 to line 3 and 4 (as recommended in previous review) this should be enough to fix the linter error (now that https://github.com/ament/ament_lint/pull/106 has been merged)